pkg/kev/config/defaults.go

```diff
@@ -45,13 +45,13 @@ const (
 	DefaultImagePullSecret = ""
 
 	// DefaultSecurityContextRunAsUser default UID for pod security context
-	DefaultSecurityContextRunAsUser = ""
+	DefaultSecurityContextRunAsUser = 0
```

Comment:
We only want to set security context details if provided. In this case `0` represent a valid value which means the default security context will be set wrongly.